src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -49,6 +49,12 @@
   - name: Gets health state of the environment, without stopping on first error.
     text: >
         az acr check-health --ignore-errors
+  - name: Verify that blobs within a given image 'sha256:abc123' can be downloaded from the repository.
+    text: >
+        az acr check-health -n MyRegistry -r MyRepository --image sha256:abc123
```

Comment:
why is only image using the long name? isn't -t the same?

Reply:
my point is if you already support -t then we should use that in the example instead of the longer nameing.

Reply:
@rosanch Yes, there is already an example using '-t' below it
![image](https://github.com/Azure/azure-cli/assets/82487617/9fe32646-6cd8-427e-8a86-484e52347eaa)